app/main.py

```diff
@@ -2,0 +1,19 @@
+from typing import List
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: List[Elf]) -> None:
```

Comment:
The type hint `List[Elf]` assumes that `Elf` is a class or type. Ensure that `Elf` is correctly defined in `app.players.elves.elf` and is intended to be used as a type hint here.